connector/src/main/java/org/geysermc/connector/network/translators/item/RecipeRegistry.java

```diff
@@ -97,11 +124,39 @@
      */
     private static CraftingData getCraftingDataFromJsonNode(JsonNode node) {
         ItemData output = ItemRegistry.getBedrockItemFromJson(node.get("output").get(0));
+        UUID uuid = UUID.randomUUID();
+        if (node.get("type").asInt() == 1) {
+            // Shaped recipe
+            List<String> shape = new ArrayList<>();
+            // Get the shape of the recipe
```

Comment:
Can the comments in this section get blank lines above them to break this section of the code up a bit?